docs/developers/guides/gas/gas-fees.mdx

```diff
@@ -36,12 +36,67 @@ and gas used.
 `linea_estimateGas` is the recommended method for estimating gas on Linea. See our 
 [reference page](../../reference/api/linea-estimategas.mdx) for more information. 
 
-Linea also supports [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas),
-[`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice), and
-[`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+Linea also supports:
+- [`eth_estimateGas`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_estimategas)
+- [`eth_gasPrice`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_gasprice)
+- [`eth_feeHistory`](https://docs.infura.io/api/networks/linea/json-rpc-methods/eth_feehistory).
+
+## Gas pricing
+
+The gas price you retrieve using methods such as `linea_estimateGas` is based on values calculated
+by the [coordinator](../../../architecture/stack/coordinator/index.mdx), and uses the gas price of 
+the previous block with a profitability multiplier applied.
+
+Firstly, the coordinator records:
+- A fixed cost of 0.03 Gwei, which reflects infrastructure costs, and;
+- Variable cost, which is the cost per byte of data submitted to L1. 
+
+These values are sent to the sequencer so that it can compute the gas prices for the upcoming 
+block and ensure that included transactions are profitable.
+
+Variable cost is calculated with the following formula:
+
+```python
+VARIABLE_COST (4 bytes) = min(max(
+(
+  (
+    ((averageWeightedBaseFee + averageWeigthedPriorityFee) * 
+    blob-submission-expected-execution-gas + averageWeightedBlobBaseFee * expected-blob-gas
+  ) / bytes-per-data-submission) * profit-margin
+)
+, min-bound), max-bound)
+```
+
+The `profit-margin` is `5`, ensuring sustainable network profitability. `min-bound` is `0.050000000` 
```

Comment:
It's set to 3, not 5. But I'm not sure if we should to the emphasis on this publically

Reply:
Min and max bounds for variable cost are also inaccurate. And they're subject to change